catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/OperationsSecuritySupport.java

```diff
@@ -37,4 +43,30 @@ void buildPolicyMap(HashMap<String, Set<String>> policyMap,
             }
         }
     }
+
+    /**
+     * Creates and returns a map of properties containing a security subject. If the correct subject
+     * type can not be obtained from the passed in subject or from the thread context no subject will
+     * be added to the properties.
+     * If the passed in subject is not compatible the method will fallback to trying to pull it from
+     * the thread context.
+     * @param subject The subject to be added. Can be {@code null}
+     */
+    Map<String, Serializable> getPropertiesWithSubject(Object subject) {
```

Comment:
This method feels odd. It seems that while callers might want a `Map<String, Serializable>` that you're contorting yourself here to provide that. I'd think just returning the subject or null would be sufficient. I might be missing something this early in the review, however.